src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -125,6 +131,21 @@ export default () => {
   });
   const assets = assetsData?.assets.nodes.filter(Boolean) as AssetNodes;
 
+  const [createAssetMutation] = useCreateAssetMutation();
```

Comment:
Probably a standard we need to discuss and start using across the app, but for now in this file it seems all the useXXXXMutation() declarations are put together above, so if you could move this up (or move the above declarations to just above where they are used(like how you wrote here))